docs/semgrep-app/findings.md

```diff
@@ -46,41 +65,47 @@ The Semgrep App Finding page displays findings across all projects connected to
 1. Sign in to Semgrep App.
```

Comment:
Commenting on line above (related issue to why I comment here: https://github.com/isaacs/github/issues/1655)

Blocking: Change `## Viewing data in the Findings page` to `## Viewing data on the Findings page`